docker-compose.test.yml

```
@@ -12,7 +12,7 @@ services:
     ports:
       - '5433:5432'
     healthcheck:
-      test: ['CMD-SHELL', 'pg_isready -U testuser -d bluelight_test']
+      test: [ 'CMD-SHELL', 'pg_isready -U testuser -d bluelight_test' ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)